zdm/tests/Performance/profile_linear.py

```diff
@@ -0,0 +1,126 @@
+""" Time profiling """
+
+######
+# first run this to generate surveys and parameter sets, by 
```

Comment:
Add comment about the expected output: is this compute time in seconds for each method?